packages/viewer/src/app/Test.svelte

```diff
@@ -6,11 +6,15 @@
 
   import type { EmbeddingAtlasProps } from "../api.js";
   import { initializeDatabase } from "../utils/database.js";
+  import { downloadBuffer } from "../utils/download";
+  import { exportMosaicSelection, type ExportFormat } from "../utils/mosaic_exporter";
```

Comment:
```suggestion
  import { downloadBuffer } from "../utils/download.js";
  import { exportMosaicSelection, type ExportFormat } from "../utils/mosaic_exporter.js";
```

for consistency with existing imports.